x-pack/plugins/apm/public/store/urlParams.ts

```diff
@@ -20,6 +19,16 @@ import { IReduxState } from './rootReducer';
 // ACTION TYPES
 export const TIMEPICKER_UPDATE = 'TIMEPICKER_UPDATE';
 
+interface LocationAction {
+  type: typeof LOCATION_UPDATE;
```

Comment:
I initially expected this to be type "string", but it works out to the actual string value which is great. Good find :+1:

Reply:
Yes, it's pretty magic :D